inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditor/animationCurveEditorComponent.tsx

```diff
@@ -61,7 +61,7 @@ export class AnimationCurveEditorComponent extends React.Component<
     private _onKeyDown(evt: KeyboardEvent) {
         switch (evt.key) {
             case "Delete":
-                if (this.props.context.activeKeyPoints?.length) {
+                if (this.props.context.activeKeyPoints?.length && !this.props.context.focusedInput) {

```

Comment:
can you explain why we need this test?

Reply:
I noticed that if I selected a key and tried to use the Delete key for changing the value, the key was deleted, so the test is to avoid this situation.